chapter_11_ansible.asciidoc

```diff
@@ -690,6 +690,13 @@ false, "AttachStdout": true, "Cmd": ["gunicorn", "--bind", ":8888",
 // check their user has permission.
 // DAVID: by the way it's getting annoying having to type my sudo password in all the time.
 
+// ZIPPERER: I got error
+// TASK [Export container image locally]
+// ... ModuleNotFoundError: No module named 'requests'
+// "Failed to import the required Python library (requests) on <local hostname>'s Python ../.venv/bin/python3.
+//
+// No error after I installed library `requests` in venv (i.e. `python3 -m pip install requests`) and reran the `ansible-playbook` command.
```

Comment:
I was only able to reproduce this error when I ran `pip install ansible` but skipped pip-installing `docker`. `docker` pulls `requests` as a dependency:
https://github.com/docker/docker-py/blob/main/pyproject.toml#L33

 Did you pip-install both ansible and docker?

Reply:
Thank you for reproducing this! Yes, this is my fault. I `python3 -m pip install ansible`, and I did not `python3 -m pip install docker`. 

I'll remove the comment this commit adds.

When I read the section [Installing ansible](https://www.obeythetestinggoat.com/book/chapter_11_ansible.html#_installing_ansible) on the site, 
![20240512-installing-ansible-section](https://github.com/hjwp/Book-TDD-Web-Dev-Python/assets/47086307/5f7789c7-cd8a-4c15-8f65-755219f5d41e)
I installed only ansible. 

I should have checked in [the current version of the source](https://github.com/hjwp/Book-TDD-Web-Dev-Python/blob/main/chapter_11_ansible.asciidoc?plain=1#L320-L322) to see the additional step to `python3 -m pip install docker`.

Reply:
the latest instructions do have an explicit `pip install docker` as well.  let me publish them to the public site...